docs/src/main/asciidoc/native-reference.adoc

```diff
@@ -206,7 +270,16 @@ These reports are some of the most useful resources when encountering issues wit
 This is useful for getting an approximation on why a method/class is included in the binary.
 However, the text format makes it very difficult to read and can take up a lot of space.
 
-Since Mandrel 21.3.0.0-Final, the call tree is also reported as a group of CSV files.
+Since Mandrel 21.3.0.0, the call tree is also reported as a group of CSV files.
+The CSV output can be enabled by adding `-H:PrintAnalysisCallTreeType=CSV` to the additional native arguments. E.g.
+
+[source,bash,subs=attributes+]
+----
+./mvnw package -DskipTests -Pnative \
+    -Dquarkus.native.enable-reports \
```

Comment:
```suggestion
```
`-Dquarkus.native.enable-reports` (i.e. `-H:PrintAnalysisCallTree`) is implied by `-H:PrintAnalysisCallTreeType=CSV`.

Reply:
Hmmm, but the user might not be aware of this subtlety. We can't expect them to be aware of the secondary effect that advanced options have. I'd rather be explicit here and keep `enable-reports`.

Side note, the need for the extra option will eventually go away https://github.com/quarkusio/quarkus/issues/20733. I'll tackle that next few days now that `main` is on 22.0.